### DISCUSSION TITLE
默认的输入框增强，点击三次空格键自动翻译，对于不知情的用户太容易误触了！

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [immersive-translate/immersive-translate#908] [General]
我经常在Github Wiki上面进行编辑，经常需要表格排版或者yaml排版，需要连续输入空格。但是有一天在一篇英文文档中长按空格的时候，输入了几个空格就突然开始转圈圈，然后把我之前的输入打断。

我一开始不知道是怎么回事。之后一直都是在记事本里面打好空格，复制粘贴进来，这样大概用了有几个月，非常烦人。

后来我在编辑一篇中文文档的时候，这一行为又出现了，并把我的文字翻译成英文。我确定是某个软件提供的翻译功能，[但是网上仍然搜不到](https://www.google.com/search?q=github%E9%95%BF%E6%8C%89%E7%A9%BA%E6%A0%BC%E8%87%AA%E5%8A%A8%E7%BF%BB%E8%AF%91)，

<img width="517" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/54425948/8e21c3df-3636-4ba0-aec4-a754e9cbb274">

后来今天终于有时间想看看到底是怎么回事，就一个一个插件关闭，逐个排查（装了几十个插件），最终才找到是immersive-translate插件，然后才在设置项里面找到关闭的方法。

其实找到关闭方法之后回过头来看，这个设计貌似很合理，可以在设置项里面打开和关闭，对于有需要的用户来说很方便。但是我之前一直在不知情的情况下用着，很难把这一行为和“immersive-translate插件”，“翻译”，以及“快速点击三次空格”这个具体的触发条件联系起来，思维总是往错误，甚至离谱的方向想。我甚至还想过是不是Github和Edge的功能，差点去给它们提交反馈了。

**Comment by DUWENINK** [comment]
同样的问题 我想起了微软的一个臭名昭著的设置 粘滞键
